src/api/reports/summary_report.py

```diff
@@ -203,125 +253,109 @@ def get_viewset_csv_content(view_cls, project_pk, request):
     return filtered_rows
 
 
-def write_data(wb, sheet_name, data):
-    casted_data = []
-    for row in data:
-        casted_row = [cast_str_value(col) for col in row]
-        casted_data.append(casted_row)
+def _get_project_metadata(project_ids):
+    projects = Project.objects.filter(pk__in=project_ids)
+    prj_serializer = ProjectCSVSerializer(projects, show_display_fields=True)
+    header = [f.display for f in prj_serializer.fields]
+    return [header] + [r.values() for r in prj_serializer.data]
 
-    ws = wb.new_sheet(sheet_name, data=casted_data)
-    ws.set_row_style(1, Style(font=Font(bold=True)))
 
+def create_benthic_lit_report(request, project_ids, data_policy_level=Project.PRIVATE):
+    return create_protocol_report(request, project_ids, BENTHICLIT_PROTOCOL, data_policy_level)
 
-@timing
-def _create_report(request, project_pk, views, sheet_names):
-    wb = Workbook()
-    for view, sheet_name in zip(views, sheet_names):
-        content = get_viewset_csv_content(view, project_pk, request)
-        write_data(wb, sheet_name, content)
 
-    return wb
+def create_benthic_pit_report(request, project_ids, data_policy_level=Project.PRIVATE):
```

Comment:
Again not requesting a change, just curious whether flattening this (i.e. including the protocol argument in a direct call to `create_protocol_report`) would be more concise